react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleSubmit(event) {
+    console.log('state:', this.state.value);
+    event.preventDefault();
+  }
+
+  render() {
+    return <form onSubmit={this.handleSubmit}>
+      <label>
+        Email:
+        <input type="email" onChange={this.handleChange}></input>
+      </label>
+      <button>Sign Up</button>
+    </form>;
```

Comment:
Although this usually works fine, it is best practice to wrap your elements in parentheses `( )`;